content/en/ensign/index.md

```diff
@@ -1,20 +1,38 @@
 ---
 date: 2022-12-15T10:09:42Z
-title: "Ensign for Data Teams"
-headertext: "Data Science 2.0"
-subheadertext: "Streaming Predictions for Realtime Models"
+title: "Ensign Use Cases"
+headertext: "Ensign Use Cases"
 noicon: true
 type: "ensign"
 switch:
     link: /ensign-enterprise/
     text: "Learn about Ensign for Enterprise"
 ---
 
-## Think Outside the Batch
+## Use Case 1: Turn Your Model into an API
+Data science teams are realizing that they must be in charge of deploying and managing their models. Sound controversial? Check out [this article](https://venturebeat.com/ai/why-do-87-of-data-science-projects-never-make-it-into-production/) about why 87% of data science projects never make it to production.
 
-Batches are great, but **the world is moving towards bigger training datasets and faster analytics** that need event streams. Even though Kafka is really powerful, most of us still don't feel like eventing is for us. :confused:
+Using [Ensign](https://rotational.app/) to turn your model into an API is the easiest way for a data science team to get a trained model into production. Ensign can be used to register requests for predictions from multiple applications, to route those inputs to the trained model, and to return model predictions back to the user.
+
+{{< figure src="/img/ensign/model_api_use_case.png" alt="A high-level illustation that shows how Ensign can be used to register requests for predictions from multiple applications, to route inputs to the model, and to route model predictions back to the user." >}}
+
+
+## Use Case 2: Bootstrap an LLM with Transfer Learning
+
+Large language models (LLMs) are powerful but tricky. What if blackbox models like ChatGPT don't work for your use case? What if you don't have enough data to train a model from scratch?
+
+Using [Ensign](https://rotational.app/) and HuggingFace, you can start incrementally delivering insights to your organization **now**. Ensign can be used to ingest training data for transfer learning and to route that training data either to a Hugging Face model trainer or the bootstrapped Hugging Face model (or to both).
```

Comment:
Should we also include a link to the blogpost about HuggingFace and Ensign? https://rotational.io/blog/streaming-nlp-with-llms-and-ensign/